test/optimizador_semanal.test.ts

```diff
@@ -0,0 +1,76 @@
+// Tests
+
+import { OptimizadorSemanal } from '../src/optimizador_semanal';
+import { Archivo } from '../src/archivo';
+
+describe('OptimizadorSemanal', () => {
+    it('debería crear un horario con actividades fijas', () => {
+        const optimizador = new OptimizadorSemanal();
+        const horario = new Archivo("./data/horario.txt");
+
+        optimizador.extraerHorario(horario);
+
+        optimizador.organizarHorario();
+
+        const expectedHorario = [
```

Comment:
Como os comenté, no debéis comparar el resultado con lo que uno espera que dé una función específica. De [los errores frecuentes](https://github.com/JJ/IV-/blob/master/errores/errores-objetivo-4.md#sobre-la-programaci%C3%B3n):

> El error principal es testear que las funciones dan la salida esperada, es decir, aplicar la lógica de negocio y si uno espera que el resultado sea X, comprobar que es X. En realidad, no es así. Hay que comprobar que se cumple lo que pide la lógica de negocio. Si se pide que algo sea mínimo, o mejor, u óptimo, hay que comprobar que efectivamente el resultado es el mínimo, o mejor que otros, u óptimo; hay que tener una función que compruebe la "bondad" o lo que sea de la solución y mostrar que efectivamente es la mejor según ese criterio. Esto es repetir lo de arriba, pero hay que tenerlo en cuenta a la hora de escribir los tests. En tu HU #2 dices:

> Quiero una forma de optimizar mi tiempo para poder equilibrar mis estudios, el ejercicio físico y las citas médicas sin sentirme abrumada.

Lo que tienes, primero, es definir qué significa "equilibrado" (¿por día? ¿por semana? ¿bien repartido a lo largo de la semana?) y a continuación crear un test que efectivamente compruebe que tu solución optimiza el tiempo y está equilibrada.